homeassistant/components/wittiot/config_flow.py

```diff
@@ -0,0 +1,59 @@
+"""Config flow for WittIOT integration."""
+from __future__ import annotations
+
+import logging
+
+import voluptuous as vol
+from wittiot import API
+from wittiot.errors import WittiotError
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers import aiohttp_client
+
+from .const import CONNECTION_TYPE, DEVICE_NAME, DOMAIN, LOCAL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for WittIOT."""
+
+    async def async_step_user(self, user_input=None):
+        """Handle the local step."""
+        errors = {}
+        data_schema = vol.Schema({vol.Required(CONF_HOST): str})
+
+        if user_input is not None:
+            ip = user_input[CONF_HOST]
+
+            api = API(ip, session=aiohttp_client.async_get_clientsession(self.hass))
+
+            try:
+                devices = await api.request_loc_info()
+            except WittiotError:
+                errors["base"] = "cannot_connect"
+            _LOGGER.debug("New data received: %s", devices)
+
+            if not devices:
+                errors["base"] = "cannot_connect"
+
+            if not errors:
+                unique_id = devices["dev_name"]
```

Comment:
Can the `dev_name` change?

Reply:
'dev_name' will not be changed

Reply:
Will or can

Reply:
I mean, the name of each device is unique. Cannot be changed.